pentaho-js/src/main/javascript/cdf-legacy/components/Pentaho.Reporting.js

```diff
@@ -918,13 +918,20 @@ var SchedulePrptComponent = PrptComponent.extend({
       }
       parameters["jobParameters"] = jobParameters;
       var success = false;
+      var protectedUrl = wd.cdf.endpoints.getWebsocketWebapp() + "/api/scheduler/job";
+      var csrfToken = pho.csrfUtil.getToken(protectedUrl);
+      var headers = {};
+      if(csrfToken !== null) {
+          headers[csrfToken.header] = csrfToken.token;
+      }
       var x = $.ajaxSettings.async;
       $.ajaxSetup({
           async: false
       });
       $.ajax({
         url: wd.cdf.endpoints.getScheduledJob(),
         type: "POST",
+        headers: headers,
```

Comment:
This should be receiving as `url` the exact same value used to obtain the token, above, the value of the `protectedUrl` variable.